website/templates/project/settings.mako

```diff
@@ -11,17 +11,81 @@
 ##        }
 ##    }'></div>
 
+<!-- The following jquery functions are to fix the bootstrap affix issue
```

Comment:
Could you [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) this code up? 

Wrapping all of this in a function that accepts the target DOM element's id (and perhaps also configuration options -- e.g. offsets -- but that's not a blocker now as it's currently the same between both uses), and putting that in `osfHelpers.js` would probably be best. That way, you could just import and instantiate it; something like 
```js
var StickyElement = require('js/osfHelpers').StickyElement;
new StickyElement('#whicheversettingspanel'); 
```
should do all of this work.